FirebaseFunctions/Tests/Integration/IntegrationTests.swift

```diff
@@ -83,7 +83,7 @@ class IntegrationTests: XCTestCase {
     return URL(string: "http://localhost:5005/functions-integration-test/us-central1/\(funcName)")!
   }
 
-  func testData() {
+  @MainActor func testData() {
```

Comment:
Why do the tests need to run on `@MainActor`?

Reply:
The `waitForExpectations` API is now labeled with `@MainActor`, so we either need to mark the class main actor, the test method main actor, or convert the test method to only use the async version.

<img width="2030" alt="Screenshot 2025-05-12 at 6 22 40 PM" src="https://github.com/user-attachments/assets/55e9ab7f-4f03-4659-882c-3e0d7b4fbb99" />

Reply:
Labeling on a per-method basis seemed lightweight. I wouldn't be opposed to phasing out tests that use completion handler based API variants and instead only build testing them in the API build tests.